src/terminal/Sequencer.cpp

```diff
@@ -1363,7 +1363,10 @@ ApplyResult Sequencer<T>::apply(FunctionDefinition const& _function, Sequence co
     case CUU: screen_.moveCursorUp(_seq.param_or<LineCount>(0, LineCount { 1 })); break;
     case DA1: screen_.sendDeviceAttributes(); break;
     case DA2: screen_.sendTerminalId(); break;
-    case DA3: return ApplyResult::Unsupported;
+    case DA3:
+        // terminal identification, 4 hex codes
+        screen_.reply("\033PC0000000\033\\");
+        break;
```

Comment:
FYI the `DECRPTUI` response sequence is identified by the characters `!|` (see https://vt100.net/docs/vt510-rm/DA3.html). So what you need here is something like this:

`"\033P!|C0000000\033\\"`

That's assuming you want `C0000000` as your "unit id".

Reply:
damn, I should not code while watching TV. Thanks @j4james, and a happy new year. ;-)

Reply:
Personally I really dislike `DA3`, but it seems to be queried by notcurses, so I thought it might still be good to have one, even though notcurses is using the (IMHO better) `XTVERSION`, which replies with terminal string (and version) string.

Reply:
Happy New Year to you too!